SecurityCenter/.repo-metadata.json

```diff
@@ -1,7 +1,7 @@
 {
     "language": "php",
     "distribution_name": "google/cloud-security-center",
-    "release_level": "stable",
```

Comment:
Seems weird. I've asked clarification regarding this internally. 

Please wait until we clarify the behaviour.

Reply:
As we discussed internally, its a preview, not beta. Since we already have the stable V2 in place, we will provide `release_level` as `stable`.

Reply:
Updated the PR to only contain the `V2` changes as thats the standard we follow in case of owlbot PRs.